docs/policy/topology-aware.md

```diff
@@ -369,3 +369,22 @@ the amount of memory for containers in `Burstable` pods or run the
 [resource-annotating webhook](../webhook.md) to provide `cri-resmgr` with
 an exact copy of the resource requirements from the Pod Spec as an extra
 Pod annotation.
+
+## Reserved pool namespaces
+
+User is able to mark certain namespaces to have reserved CPU allocation.
+The `ReservedPoolNamespaces` option is a list of namespace globs that will be
+allocated to reserved CPU class.
+
+For example:
+
+```yaml
+policy:
+  Active: topology-aware
+  topology-aware:
+    ReservedPoolNamespaces: ["my-pool","reserved-*","kube-system"]
```

Comment:
I think it'd be better to leave out `kube-system` here, because as mentioned later it is implicitly always a reserved pool.